webui/module/Application/config/commands.csv

```diff
@@ -1,83 +1,83 @@
-setbandwidth,--,optional,--,--,--,--,--,--,--,--,--,--
-setdebug,--,optional,--,--,--,--,--,--,--,--,--,--
-setip,--,optional,--,--,--,--,--,--,--,--,--,--
-show,--,optional,--,--,--,--,--,--,--,optional,--,--
-sqlquery,--,optional,--,--,--,--,--,--,--,--,--,--
-time,--,optional,--,--,--,--,--,--,--,--,--,--
-trace,--,optional,--,--,--,--,--,--,--,--,--,--
-truncate,--,optional,--,--,--,--,--,--,--,--,--,--
-unmount,--,optional,--,--,--,--,--,--,--,--,--,--
-umount,--,optional,--,--,--,--,--,--,--,--,--,--
-update,--,optional,--,--,--,--,--,--,--,--,optional,--
-use,required,required,required,required,required,required,required,required,required,required,required,required
-var,--,optional,--,--,--,--,--,--,--,--,--,--
-version,required,optional,--,--,--,--,--,--,--,--,--,--
-wait,--,optional,--,--,--,--,--,--,--,--,--,--
-whoami,--,optional,--,--,--,--,--,--,--,--,--,--
-.actiononpurge,--,optional,--,--,--,--,--,--,--,--,--,--
-.api,required,required,required,required,required,required,required,required,required,required,required,required
-.clients,required,optional,--,--,--,--,--,--,--,--,--,--
-.catalogs,--,optional,--,--,--,--,--,--,--,--,--,--
-.defaults,--,optional,--,--,--,required,--,--,--,--,--,--
-.filesets,--,optional,--,--,--,required,--,--,required,--,--,--
-.help,required,required,required,required,required,required,required,required,required,required,required,required
-.jobdefs,--,optional,--,--,--,--,--,--,--,--,--,--
-.jobs,--,optional,--,--,--,required,--,--,required,--,--,--
-.levels,--,optional,--,--,--,--,--,--,--,--,--,--
-.locations,--,optional,--,--,--,--,--,--,--,--,--,--
-.messages,--,optional,--,--,--,--,--,--,--,--,--,--
-.media,--,optional,--,--,--,--,--,--,--,--,--,--
-.mediatypes,--,optional,--,--,--,--,--,--,--,--,--,--
-.msgs,--,optional,--,--,--,--,--,--,--,--,--,--
-.pools,--,optional,--,--,--,required,--,--,--,--,optional,--
-.profiles,--,optional,--,--,--,--,--,--,--,--,--,--
-.quit,--,optional,--,--,--,--,--,--,--,--,--,--
-.sql,--,optional,--,--,--,--,--,--,--,--,--,--
-.schedule,--,optional,--,--,--,--,--,--,--,required,--,--
-.status,--,optional,--,--,--,--,--,--,--,--,--,--
-.storages,--,optional,--,--,--,required,--,--,--,--,--,--
-.types,--,optional,--,--,--,--,--,--,--,--,--,--
-.volstatus,--,optional,--,--,--,--,--,--,--,--,--,--
-.bvfs_lsdirs,--,optional,--,--,--,--,--,--,required,--,--,--
-.bvfs_lsfiles,--,optional,--,--,--,--,--,--,required,--,--,--
-.bvfs_update,--,optional,--,--,--,--,--,--,required,--,--,--
-.bvfs_get_jobids,--,optional,--,--,--,--,--,--,required,--,--,--
-.bvfs_versions,--,optional,--,--,--,--,--,--,required,--,--,--
-.bvfs_restore,--,optional,--,--,--,--,--,--,required,--,--,--
-.bvfs_cleanup,--,optional,--,--,--,--,--,--,--,--,--,--
-.bvfs_clear_cache,--,optional,--,--,--,--,--,--,--,--,--,--
+Command,Client,Console,Dashboard,Director,Fileset,Job,Media,Pool,Restore,Schedule,Storage,Analytics,Subscription
```

Comment:
Is the new Subscription column still required/useful?

Reply:
This is outdate, has already been removed.